scijava-ops-engine/src/main/java/org/scijava/ops/engine/MatchingConditions.java

```diff
@@ -70,4 +70,9 @@ public int hashCode() {
 		return Objects.hash(request(), hints());
 	}
 
+	@Override
+	public String toString() {
+		return request.toString();
+	}
```

Comment:
Forgot I included this change, but it is nice!

Reply:
Maybe we should put this and the `helpVerbose()` change (and other nice small changes) in their own PRs?

Reply:
Added it to #129